app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :user
-  belongs_to :comment
+  belongs_to :post
+
+  def self.update_comments_counter(post_id)
+    Post.increment_counter(:comments_counter, post_id)
+  end
 end
```

Comment:
- [ ] Again here it would be nice to use one of the Rails Active Callbacks - after_save to call the #update_comments_counter method every time a new comment has been saved to database,